release/kernel/README.md

```diff
@@ -13,9 +13,9 @@ FROM %%IMAGE%%:kernel
 COPY server.xml /config/
 ```
 
-The `microProfile1` image contains the features required to implement Eclipse MicroProfile 1.2. The `webProfile7` image contains the features required for Java EE7 Web Profile compliance. The `javaee7` image adds the features required for Java EE7 Full Platform compliance. The `javaee7` image is also tagged with `latest`.
+The `microProfile1` image contains the features required to implement Eclipse MicroProfile 1.2. The `webProfile8` image contains the features required for Java EE8 Web Profile compliance. The `javaee8` image adds the features required for Java EE8 Full Platform compliance. The `javaee8` image is also tagged with `latest`. The `webProfile7` image contains the features required for Java EE7 Web Profile compliance. The `javaee7` image adds the features required for Java EE7 Full Platform compliance.
```

Comment:
I think the microProfile1 image does MicroProfile 1.3, since we are updating it would be good to update this to be correct.